nss-evening-cohort-26/pos-terminal-the-algorithm-avengers
# Order Items can now be viewed in the Orders

## Description

- Created api calls to get items, single items, create items, update items and delete items in itemData.js.
- Added api call to get a single order in orderData.js
- Created a merged api call to get orders and its items in mergedData.js
- Created an api call to get order items that have the ids for the order and item in orderItemsData.js
- Added an event to domEvents.js to allow the items of an order to show when details is clicked
- Made total show the total of the order based on all the items price
- Add minimal styling

## Related Issue
#12 

## Motivation and Context
This is necessary to allow users to view the items in their order when the details button is clicked and to see the total of their order.

## How Can This Be Tested?
To test this PR, please do the follow in your VSCode in terminal:
$ git fetch origin teammates-branch-name 
$ git checkout teammates-branch-name

PLEASE go to FireBase and edit orderItems and add the item_id (firebaseKey of items) that is related to your uid. In orderItems currently, the item_id has the first letter of your name so you will know which ones is yours. If this is not done, you will get errors in your console. See example screenshot below.

## Screenshots (if appropriate):
![image](https://github.com/nss-evening-cohort-26/pos-terminal-the-algorithm-avengers/assets/153968439/6a5c8869-3dc9-4599-b454-c0bb314d3db3)


## Types of changes
